### ISSUE TITLE
meshery-perf crashing

### ISSUE THREAD

**Issue by acald-creator** [meshery/meshery-perf#8]
#### Current Behavior

CrashLoopBackOff

#### Expected Behavior

Using `mesheryctl` to install on the `k3d` cluster, should run perfectly fine, but both `meshery-perf` and `meshery-cillium` is crashing. 

Reference to `cilium-adapter` crashing, [#71](https://github.com/meshery/meshery-cilium/issues/71)

#### Screenshots/Logs

```bash
kubectl logs meshery-perf-794bb4f4c7-4tf6l -n meshery
time="2022-09-05T22:20:51Z" level=info msg="Component listening at port: 10013" app=meshery-perf
time="2022-09-05T22:20:51Z" level=info msg="Registering static workloads with Meshery Server..." app=meshery-perf
time="2022-09-05T22:20:51Z" level=info msg="Registering latest components with Meshery Server" app=meshery-perf
time="2022-09-05T22:20:51Z" level=info msg="Registering latest workload components for version " app=meshery-perf
time="2022-09-05T22:20:51Z" level=info msg="Failed to generate components for version " app=meshery-perf
time="2022-09-05T22:20:51Z" level=error msg="mkdir /templates/oam/workloads: no such file or directory" app=meshery-perf code=1013 probable-cause="Invalid Path or version passed in static configuration.URL passed maybe incorrect.Version passed maybe incorrect" severity=2 short-description="error creating components" suggested-remediation="Make sure to pass correct configuration.Make sure the URL passed in the configuration is correct.Make sure a valid version is passed in configuration"
panic: interface conversion: error is *fs.PathError, not *errors.Error

goroutine 41 [running]:
github.com/layer5io/meshkit/errors.GetCode({0x287ee80, 0xc00025e090})
        /go/pkg/mod/github.com/layer5io/meshkit@v0.5.5/errors/errors.go:90 +0x90
github.com/layer5io/meshkit/logger.(*Logger).Error(0xc000010018, {0x287ee80, 0xc00025e090})
        /go/pkg/mod/github.com/layer5io/meshkit@v0.5.5/logger/logger.go:69 +0xbb
main.registerCapabilities({0x25fa974, 0x5}, {0x28d9090, 0xc000010018})
        /build/main.go:142 +0xf5
created by main.main
        /build/main.go:98 +0x590
```

#### Environment

- **Host OS:** Ubuntu WSL2
- **Platform:** Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v
- **Meshery Adapter Version:** stable-v

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by acald-creator** [comment]
Close as this was solved by PR [#6178](https://github.com/meshery/meshery/pull/6182)

**Comment by leecalcote** [comment]
Thank you.
